Mutagen.Bethesda.Analyzers.Cli/Modules/RunAnalyzerModule.cs

```diff
@@ -10,6 +10,9 @@ protected override void Load(ContainerBuilder builder)
     {
         builder.RegisterDecorator<MinimumSeverityFilter, IReportDropbox>();
         builder.RegisterDecorator<SeverityAdjuster, IReportDropbox>();
+        builder.RegisterDecorator<CsvDropbox, IReportDropbox>();
+        builder.RegisterDecorator<TopicListJoin, IReportDropbox>();
+        builder.RegisterDecorator<TopicEnricher, IReportDropbox>();
```

Comment:
Let's back off the TopicListJoin/TopicEnricher card/issue for now and do in another PR